src/wasm/wat-parser-internal.h

```diff
@@ -482,8 +626,25 @@ struct IdTok {
   friend bool operator==(const IdTok&, const IdTok&) { return true; }
 };
 
+struct StringTok {
+  std::optional<std::string> str;
```

Comment:
Is this only set when it contains escaped chars? If so that's not very obvious here... how about `escapedStr` or something, and add a comment that this `nullopt` when there's no escaped chars?

Reply:
Sure, I'll update this and `building` in a follow-up.